### PR TITLE
Updated with loading bar

### LINKED ISSUE #4: Add a loading animation
The EC2 pricing data takes a while to load. To keep the user busy, we may show a loading animation such as a spinner?

I'm open to new ideas.

### PR DESCRIPTION
**Pull request by Rud156**
Fixes #4

### CODE REVIEW

**Comment by recrsn** [comment]
@Rud156 Thanks for your PR. However, this causes the pages to jump, can you fix that ?

**Comment by Rud156** [comment]
Yup sure.

**Comment by Rud156** [comment]
@agathver Fixed and Improved

File: app/scripts/main.js [hunk 1]
```diff
@@ -179,6 +179,7 @@ $(() => {
   const $currency = $('#currency-select');
 
   function loadData(data) {
+  document.getElementById('loading-bar').style.display = 'none';
```
Comment (recrsn) [inline app/scripts/main.js]: Could use jQuery here for consistency?

Comment (Rud156) [reply app/scripts/main.js]: Ok. I'll update it

**Comment by Rud156** [comment]
@agathver Updated with jquery

**Pull request closed without merging**